src/Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml

```diff
@@ -2360,41 +2360,29 @@
 			<SolidColorBrush x:Key="SystemControlFocusVisualSecondaryBrush" Color="{ThemeResource SystemAltMediumColor}" />
 
 			<!-- SplitView, NavigationView -->
-			<win:RevealBackgroundBrush x:Key="SystemControlBackgroundBaseLowRevealBackgroundBrush" TargetTheme="Light" Color="{StaticResource SystemRevealBaseLowColor}" FallbackColor="{StaticResource SystemBaseLowColor}" />
-			<xamarin:SolidColorBrush x:Key="SystemControlBackgroundBaseLowRevealBackgroundBrush" Color="{StaticResource SystemBaseLowColor}" />
+			<SolidColorBrush x:Key="SystemControlBackgroundBaseLowRevealBackgroundBrush" Color="{ThemeResource SystemColorButtonFaceColor}" />
```

Comment:
Oh dear. I, uh... I trust you Martin. 😛

Reply:
![image](https://user-images.githubusercontent.com/1075116/93589120-a966d000-f9ac-11ea-9792-55e846d033ea.png)